1000-accordion-design-guidance.md

```diff
@@ -0,0 +1,29 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-15
+updated: 2024-10-22
+---
+
+# Accordion Design Guidance
+
+An [accordion](https://clarity.design/documentation/accordion) is a collection of vertically stacked sections with multiple content panels.
+
+## Guidance
+
+- **Must** use accordions when you need to display a lot of different information and the user doesn’t need it all at once.
+- **Must not** use an accordion if all the content is important to a user.
+- **Must not** use an accordion if there is very little visible content on the page.
+- **Must not** add too long information in the accordion content.
```

Comment:
```suggestion
- Design teams **must not** include overly lengthy information in the accordion content, as this can overwhelm users and diminish the effectiveness of the accordion's collapsible design.
```